### PR TITLE
krypton001 completed Subtask-1

### PR DESCRIPTION
**Pull request by krypton001**
krypton001 completed Subtask-1

### CODE REVIEW

File: Subtask-1/krypton001.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Name - Shivam katiyar
+Nick - Banarasi babu
+Homestate - Uttar pradesh
+Hometown - Varanasi
+
+I am excited to make a gf by impressing her by my coding skills...lol
```
Comment (GhostFoxSledgehammer) [inline Subtask-1/krypton001.txt:6]: bade naughty ho bemta
@Elemento24 @Lazy-Leopard @lazyp4nd4

Comment (lazyp4nd4) [reply Subtask-1/krypton001.txt:6]: Sahi baamt hai

Comment (Elemento24) [reply Subtask-1/krypton001.txt:6]: bade naughty ho bemta x2

Comment (divyanshrai655) [reply Subtask-1/krypton001.txt:6]: Sahi baamt hai  :p

**Comment by krypton001** [comment]
😶...Sirs kisi ko btaiyega mat.

**Pull request merged**